openedx_tagging/core/tagging/models.py

```diff
@@ -326,6 +326,9 @@ class ObjectTag(models.Model):
         ),
     )
 
+    class Meta:
+        index_together = ["taxonomy", "_value"]
+
```

Comment:
@ChrisChV  This change has been included in https://github.com/openedx/openedx-learning/pull/57/files#diff-204875f41064da3abfd4648611ad41c6f21abe09277263410e3daafbd343f2a3R349-R351

so can be removed from here.

Reply:
Sure, thanks :+1: